readme.md

```diff
@@ -176,6 +176,9 @@ Set the viewport dimensions
 #### .screenshot(path)
 Saves a screenshot of the current page to the specified `path`. Useful for debugging. Note the path must include the file extension. Supported formats include .png, .gif, .jpeg, and .pdf.
 
+#### .extractDomElement(selector,path)
```

Comment:
Can this be renamed to `.screenshotDomElement` ? to me extracting sounds like it's going to pull it off the page, or return the element rather than screenshot it